### ISSUE TITLE
No sensor data is present from some of the sensors

### ISSUE THREAD

**Issue by cornyJokeHere** [Limych/ha-narodmon#62]
**Environment**

- Home Assistant Core release with the issue: core-2021.11.3
- This custom component release with the issue: 1.1.5
- Last working this custom component release (if known):
- Operating environment (Home Assistant/Supervisor/Docker/venv): Supervisor / HA OS

**Describe the bug**

Some sensors work and provide data, and some don't. Tried rebooting the system, updating everything, reinstalling Narodmon component, reissuing the API key from Narodmon.

![Screenshot 2022-02-15 at 16 11 27](https://user-images.githubusercontent.com/79423979/154068847-066d5a67-601c-4a31-a4d8-86613b920560.png)

![Screenshot 2022-02-15 at 16 13 11](https://user-images.githubusercontent.com/79423979/154069034-a7db7232-1f30-4070-82e3-6da35e51a86c.png)

**Configuration.yaml**
```yaml

# Configure a default setup of Home Assistant (frontend, api, etc)
default_config:

# Text to speech
tts:
  - platform: google_translate

group: !include groups.yaml
automation: !include automations.yaml
script: !include scripts.yaml
scene: !include scenes.yaml

narodmon:
  apikey: #############
  devices:
    - name: "Narodmon"
        
input_select:
  # Xiaomi Air Purifier 2S
  xiaomi_airpurifier_mode:
    name: Mode
    options:
      - Auto
      - Silent
      - Favorite
    icon: "mdi:animation-outline"
    
logger:
  default: info
  logs:
    hacs: debug
    custom_components.hacs: debug
    queueman: debug
    aiogithubapi: debug
    custom_component.narodmon: debug
    
frontend:
  themes: !include_dir_merge_named themes

```

**Steps to Reproduce**

Install Narodmon component, try to read the data from sensors

**Expected behavior**

Sensors data is present 

**Debug log**

[home-assistant.log](https://github.com/Limych/ha-narodmon/files/8069380/home-assistant.log)

**Additional context**

**Comment by Limych** [comment]
Yes, there is. But this is not a bug, but a consequence of the fact that the Narodmon administration recently introduced limits on requests for other people's sensors.
I am currently working on a solution to this problem. How quickly it will turn out - I can not say.

As a temporary solution, you can contact the Narodmon administration and ask to increase the limits for you personally. For detailed instructions, see their website under Profile > My Apps.

**Comment by Limych** [comment]
New beta version with fix of this problem: https://github.com/Limych/ha-narodmon/tree/feature/v2.0 (only manual installation yet)

**Comment by cornyJokeHere** [comment]
Hi, thanks for your reply. Tried installing the beta, now I see this in the logs:

2022-02-20 18:16:47 ERROR (MainThread) [custom_components.narodmon] Error fetching narodmon data:
2022-02-20 18:16:59 ERROR (MainThread) [custom_components.narodmon] [403] Неверный ключ
2022-02-20 18:16:59 ERROR (MainThread) [custom_components.narodmon] Error fetching narodmon data:
2022-02-20 18:17:20 ERROR (MainThread) [custom_components.narodmon] [403] Неверный ключ

![Screenshot 2022-02-20 at 19 10 17](https://user-images.githubusercontent.com/79423979/154852210-484d85c1-ca44-46ac-bbf1-c542791e4a0b.png)
![Screenshot 2022-02-20 at 19 10 43](https://user-images.githubusercontent.com/79423979/154852239-5a88e6ce-e906-424f-8ce0-30c81f868f5c.png)

**Comment by Limych** [comment]
UPD.
I found an error. I'm trying to fix it.

**Comment by Limych** [comment]
Fixed

**Comment by Limych** [comment]
Code are merged to main branch and second beta is published: https://github.com/Limych/ha-narodmon/releases

**Comment by Limych** [comment]
Published v2.0.0
